resources/views/templates/watch/header.blade.php

```diff
@@ -0,0 +1,129 @@
+                                <img src="{{ asset('frontend/img/logo.png') }}" alt="">
+                            </a>
+                        </div>
+                    </div>
+                    <div class="col-md-6">
+                        <div class="header-search">
+
+                            {!! Form::open() !!}
+                                <select class="input-select">
+                                    <option value="0">@lang('allcategory')</option>
+                                        @for ($i = 0; $i < count($menus); $i++) 
+                                            @if ($menus[$i]->parent_id===0)
+                                                <option value="1">{!! $menus[$i]->name !!}</o>
+                                            @endif
+                                        @endfor
+                                </select>
+
+                            {{Form::text('search', trans('messages.searchhere'), ['class'=>'input'])}}
+                                {{ Form::submit(trans('messages.search'), ['class' => 'search-btn']) }}
+                            {!! Form::close() !!}
+                        </div>
+                    </div>
+                    <div class="col-md-3 clearfix">
+                        <div class="header-ctn">
+                            <!-- Cart -->
+                            <div class="dropdown">
+                                <a class="dropdown-toggle" data-toggle="dropdown" aria-expanded="true">
+                                    <i class="fa fa-shopping-cart"></i>
+                                    <span>@lang('messages.yourcart')</span>
+                                    <div class="qty">3</div>
+                                </a>
+                            </div>
+
+                            <div class="menu-toggle">
+                                <a href="#">
+                                    <i class="fa fa-bars"></i>
+                                    <span>@lang('messages.menu')</span>
+                                </a>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </header>
+
+    <nav id="navigation">
+        <div class="container">
+            <div id="responsive-nav">
+                <ul class="main-nav nav navbar-nav">
+                    <li class="active dropdown"><a href="#">@lang('messages.trangchu')</a>
+                    </li>
+                    @for ($i = 0; $i < count($menus); $i++) 
+                        @if ($menus[$i]->parent_id === 0)
+                            <li class="dropdown"><a href="#">{{ $menus[$i]->name }}</a>
+                                <div class="dropdown-content">
+                                    @for ($j = 0; $j < count($menus); $j++) 
+                                        @if ($menus[$j]->parent_id===$menus[$i]->id)
```

Comment:
@if ($menus[$j]->parent_id === $menus[$i]->id)

Reply:
ví dụ như for2->parent_id===for1->id thì in ra.

Reply:
Thêm space vào trước và sau operator chứ không cần giải thích code. Nhưng nếu giải thích code thì em cần giải thích rõ ràng chứ như này ai nhìn vào cũng biết lệnh này làm gì mà